client/app/bundles/HelloWorld/reducers/name.jsx

```diff
@@ -0,0 +1,14 @@
+import actionTypes from '../constants/helloWorldConstants';
+
+const name = (state = '', action) => {
```

Comment:
It's standard practice to write reducers with single values like that and is in the basic `Todos` example from Redux